File: contractstudy/src/test/java/test/contractstudy/extractor/abstractAnnotationExtractor/java/testData/AnnotationsMultiple.java
package test.contractstudy.extractor.abstractAnnotationExtractor.java.testData;

import javax.validation.constraints.Max;
import javax.validation.constraints.Min;
import javax.validation.constraints.NotNull;
import javax.validation.constraints.Size;

public class AnnotationsMultiple {

  @NotNull
  private String manufacturer;

  @NotNull
  @Size(min = 2, max = 14)
  private String licensePlate;

  @Min(2)
  private int seatCount;

  public AnnotationsMultiple(String manufacturer, String licencePlate, int seatCount) {
    this.manufacturer = manufacturer;
    this.licensePlate = licencePlate;
    this.seatCount = seatCount;
  }

  public String get(@Max(5) int index) {
    return "string";
  }
}

File: contractstudy/src/test/java/test/contractstudy/extractor/abstractAnnotationExtractor/kotlin/testData/AnnotationsMultiple.kt
package test.contractstudy.extractor.abstractAnnotationExtractor.kotlin.testData

import javax.validation.constraints.Max
import javax.validation.constraints.Min
import javax.validation.constraints.NotNull
import javax.validation.constraints.Size

class AnnotationsMultiple(
    @field:[NotNull] val manufacturer: String, @field:Size(
        min = 2,
        max = 14
    ) @field:NotNull val licensePlate: String, @field:Min(
        2
    ) val seatCount: Int
) {
    operator fun get(@Max(5) index: Int): String {
        return "string"
    }
}